machine_management/modifying-machineset.adoc

```diff
@@ -5,13 +5,18 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+You can modify a machine set, such as adding labels, changing the instance type, or changing block storage.
+
+On {rh-virtualization-first}, you can also migrate a machine set to a different storage domain.
```

Comment:
I would phrase this differently. This implies an automatic migration, whereas it is rather changing the machine set to provision new nodes on a new storage domain. The existing nodes are not migrated automatically.

Perhaps:

```
On {rh-virtualization-first}, you can also change a machine set to provision new nodes on a different storage domain.
```

Reply:
Sounds good to me. Fixed.